ecr/templates/auth_server_policy.json.tpl

```diff
@@ -8,7 +8,8 @@
         "AWS": [
           "arn:aws:iam::${intg_account}:role/keycloak_ecs_execution_role_intg",
           "arn:aws:iam::${staging_account}:role/keycloak_ecs_execution_role_staging",
-          "arn:aws:iam::${prod_account}:role/keycloak_ecs_execution_role_prod"
+          "arn:aws:iam::${prod_account}:role/keycloak_ecs_execution_role_prod",
+          "arn:aws:iam::${intg_account}:role/KeycloakECSExecutionRoleIntg"
```

Comment:
Assuming the `eycloak_ecs_execution_role_intg` role is still needed until the move is completed?

Will additional roles also need to be added for the other environments?

Reply:
Yep. They can't be added until they exist though so I'll have to deploy these changes, add the roles and then deploy those.